tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -107,4 +117,24 @@ SPECTRE_TEST_CASE(
       1, tmpl::list<Tags::FixedSource<Elasticity::Tags::Displacement<2>>>>(
       &BentBeamProxy::source_variables, solution, "BentBeam", {"source"},
       {{{-5., 5.}}}, std::make_tuple(), DataVector(5));
+
+  {
+    INFO("Test elasticity system with bent beam");
+    // Verify that the solution numerically solves the system
+    using system = Elasticity::FirstOrderSystem<2>;
+    const typename system::fluxes fluxes_computer{};
+    using AffineMap = domain::CoordinateMaps::Affine;
+    using AffineMap2D =
+        domain::CoordinateMaps::ProductOf2Maps<AffineMap, AffineMap>;
+    const domain::CoordinateMap<Frame::Logical, Frame::Inertial, AffineMap2D>
+        coord_map{{{-1., 1., -2.5, 2.5}, {-1., 1., -0.5, 0.5}}};
+    const Mesh<2> mesh{4, Spectral::Basis::Legendre,
+                       Spectral::Quadrature::GaussLobatto};
+    const auto logical_coords = logical_coordinates(mesh);
+    const auto system_coords = coord_map(logical_coords);
+    FirstOrderEllipticSolutionsTestHelpers::verify_solution<system>(
+        solution, fluxes_computer, mesh, coord_map,
+        std::numeric_limits<double>::epsilon() * 100.,
```

Comment:
Add a comment like "Since the solution is a polynomial of degree 2 it should numerically solve the system equations to machine precision on 3 grid points per dimension" and actually use 3 grid points per dim (you currently have 4 per dim).

Reply:
Done.